### PR TITLE
Oop homework

### PR DESCRIPTION
**Pull request by sudakevych-andrii**

### CODE REVIEW

File: oop/homework.py [hunk 1]
```diff
@@ -39,31 +39,67 @@ class Cat:
...
+        'fodder': 10,
+        'apple': 5,
+        'milk': 2
+    }
 
     def eat(self, product):
-        pass
+        product = product.lower()
+        if product in self.products_list.keys():
+            self._increase_saturation_level(self.products_list.get(product))
```
Comment (alchemixXx) [inline oop/homework.py]: make some feedback in case of bad input

File: oop/homework.py [hunk 2]
```diff
@@ -39,31 +39,67 @@ class Cat:
...
 
     def _reduce_saturation_level(self, value):
-        pass
+        if self.saturation_level < 0:
+            return 0
```
Comment (alchemixXx) [inline oop/homework.py]: you need not return value, but install self.saturation_level value 0

File: oop/homework.py [hunk 3]
```diff
@@ -39,31 +39,67 @@ class Cat:
...
+        self.saturation_level -= value
 
     def _increase_saturation_level(self, value):
-        pass
+        if self.saturation_level > 100:
+            return 100
```
Comment (alchemixXx) [inline oop/homework.py]: see comment line 59

File: oop/homework.py [hunk 4]
```diff
@@ -230,49 +298,75 @@ class House:
...
+        return int(sum([wall.number_of_rolls_of_wallpaper(roll_width_m, roll_length_m) for wall in self.__walls]))
 
     def get_room_square(self):
-        pass
+        return self.get_walls_square() - (self.get_windows_square() + self.get_door_square())
+
+
+cat1 = Cat(5)
+cat1.run(6)
+
```
Comment (alchemixXx) [inline oop/homework.py]: it's better to delete those staff in final release :)

Comment (sudakevych-andrii) [reply oop/homework.py]: Done.

File: oop/homework.py (hunk 2 above)
Comment (sudakevych-andrii) [reply oop/homework.py]: Done

File: oop/homework.py (hunk 3 above)
Comment (sudakevych-andrii) [reply oop/homework.py]: Done

File: oop/homework.py (hunk 1 above)
Comment (sudakevych-andrii) [reply oop/homework.py]: Done

**Pull request merged**